### PR TITLE
Refactoring to fullfill vendor requirements

### PR DESCRIPTION
**Pull request by randomstr1ng**
#### Descriptions:
This PR includes all the necessary changes to fullfill the WIZ.io developer docs.
Due to the amount of changes I have bumped to version 2.0.0

#### Fix:
- combined asset specific actions into single actions.
- removed obsolete actions:
  - `GET VULNERABILITIES FOR ASSET`
  - `GET ISSUES FOR ASSET`
- Updated actions with more detailed input capabilities
- Updated Queries & variables for the following actions:
  - `GET ISSUES`
  - `GET INVENTORY ASSETS`
  - `GET PROJECTS`
  - `ADD COMMENT TO ISSUE`
  - `GET VULNERABILITIES`

### ToDo:
- [ ] Regenrate Docs
- [ ] Regenerate Sample Playbooks

### CODE REVIEW

**Comment by randomstr1ng** [comment]
@cs-prerna-joshi Can you please follow up on the open ToDo's? Due to the lack of the RDK to import non-compressed reposed, I could not generate it.

File: wiz-io/release_notes.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 #### What's Improved
-- Added new action "Get Vulnerabilities for Asset" and its playbook.
-    
+- combined asset specific actions into single actions.
+- removed obsolete actions:
```
Comment (cs-renuka-andhale) [inline wiz-io/release_notes.md:3]: Please provide more details about why these actions are removed ? 
This information we require to add in release notes.

File: wiz-io/release_notes.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+  - `GET VULNERABILITIES FOR ASSET`
+  - `GET ISSUES FOR ASSET`
+- Updated actions with more detailed input capabilities
 
 #### What's Fixed
-- Updated the Query for "Get Issues" action.
+- Updated Queries for the following actions:
```
Comment (cs-renuka-andhale) [inline wiz-io/release_notes.md:9]: Please provide details about query updates.

File: wiz-io/release_notes.md (hunk 1 above)
Comment (randomstr1ng) [reply wiz-io/release_notes.md:3]: The actions have been removed as it has been noticed by the Vendor that the queires used by the two actions cause instability. The Generic Queries for Assets and Issues now include the same capabilities.

File: wiz-io/release_notes.md (hunk 2 above)
Comment (randomstr1ng) [reply wiz-io/release_notes.md:9]: The Queries now use the predefined Queries specified by the Vendor.
In addtion, it has been requested by the Vendor to not provide a "free text field" to specify queries, instead they required form fields which build the queries. The most changes have happend to the variable fields.

**Review by cs-renuka-andhale — APPROVED** [review]

**Pull request merged**